flowzone.yml

```diff
@@ -3367,6 +3493,49 @@ jobs:
           echo "sha_tag=${sha_tag}" >> "${GITHUB_OUTPUT}"
           echo "version_tag=${version_tag}" >> "${GITHUB_OUTPUT}"
 
+  cargo_sbom:
+    name: Generate SBOM for cargo
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    needs:
+      - is_cargo
+      - cargo_test
+      - versioned_source
+    if: needs.is_cargo.outputs.cargo == 'true' && needs.is_cargo.outputs.cargo_sbom == 'true'
+
+    <<: *customWorkingDirectory
+
+    steps:
+      - *getGitHubAppToken
+      - *checkoutVersionedSha
+      - *createLocalRefs
+
+      - name: Set up toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+
+      - name: Install CycloneDX for Cargo
+        run: cargo install cargo-cyclonedx
+
+      - name: Generate SBOM
+        run: |
+          cargo cyclonedx --override-filename bom
+          mv bom.xml ${{ runner.temp }}/bom.xml
+
+      - <<: *publishSBOMArtifacts
+        with:
+          name: sbom-cargo-${{ github.event.pull_request.head.sha }}
```

Comment:
Same here, we can probably drop the sha from the variable name but adding a `gh-release-` prefix might be nice to attach to the GH release.